lib/modules/manager/npm/extract/locked-versions.ts

```diff
@@ -68,7 +68,7 @@ export async function getLockedVersions(
         } else {
           packageFile.extractedConstraints!.npm = '<7';
         }
-      } else if (lockfileVersion === 2) {
+      } else if (lockfileVersion === 2 || lockfileVersion === 3) {
```

Comment:
don't we need a different condition because we need a different npm version constraint?

Reply:
@viceice i have a question regarding the extractedConstraints property. Is this only used for logging or am I missing something? package-lock v3 is backward-compatible with npm 7 so should i set '>=7' as extractedConstraints? From the npm docs:

3: The lockfile version used by npm v9. Backwards compatible to npm v7.

https://docs.npmjs.com/cli/v9/configuring-npm/package-lock-json